src/main/resources/templates/subscription.html

```diff
@@ -293,7 +308,7 @@ <h5>OR</h5>
                                 <img class="cursor-pointer" alt="Requirement Information" src="assets/images/information.png" data-toggle="tooltip" data-placement="top"
                                     title="Info: 'AND' is used between Conditions in Requirement groups, and 'OR' is used between Requirement groups."/>
                             </div>
-                            <div class="p-1 border border-primary form-group">
+                            <div class="p-1 border border-primary form-group">                            
```

Comment:
You should try to enable trim trailing white spaces in your editor, here is some trailing white spaces.